gensim/models/keyedvectors.py

```diff
@@ -154,6 +154,75 @@ def get_vector(self, entity):
         else:
             raise KeyError("'%s' not in vocabulary" % entity)
 
+    def add_entity(self, entity, weights, replace=False):
+        """Add entity vector in a manual way.
+        If `entity` is already in the vocabulary, old vector is keeped unless `replace` flag is True.
+
+        Parameters
+        ----------
+        entity : str
+            Entity specified by string tag.
+        weights : np.array
+            1D numpy array with shape (`vector_size`,)
+        replace: bool, optional
+            Boolean flag indicating whether to replace old vector if entity is already in the vocabulary.
+            Default, False, means that old vector is keeped.
+        """
+        self.add_entities([entity], weights.reshape(1, -1), replace=replace)
+
+    def add_entities(self, entities, weights, replace=False):
+        """Add entities and theirs vectors in a manual way.
+        If some entity is already in the vocabulary, old vector is keeped unless `replace` flag is True.
+
+        Parameters
+        ----------
+        entities : list of str
+            Entities specified by string tags.
+        weights: list of np.array or np.array
+            List of 1D np.array vectors or 2D np.array of vectors.
+        replace: bool, optional
+            Boolean flag indicating whether to replace vectors for entities which are already in the vocabulary.
+            Default, False, means that old vectors for those entities are keeped.
```

Comment:
No need to duplicate "default" value for trivial case in docstring, maybe better to write something like

```
Flag indicating whether to replace vectors for entities which are already in the vocabulary, if True - replace vectors, otherwise - keep old vectors.

```